scripts/dev/estimate-usage/estimate-usage.x86_64.runmode.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -e
+
+print_fs_usage() {
+    local image="$1"
+    local fs="$(mktemp)"
+    truncate -s 1G "$fs"
```

Comment:
I think you could increase this to 2G since the size is already so close to 1G. Is there an x64 target with only 1G disk space?

Reply:
Makes sense to me. Will be in next push.